nets/DevNetSpec.ts

```diff
@@ -19,6 +19,7 @@ export abstract class DevNetSpec extends NetSpec {
   readonly binary
   readonly chain
   readonly nodeCount
+  #rawChainSpecPath?: Promise<string>
```

Comment:
this can just be a string and you can await in the call

Reply:
It needs to be a promise to account for a previous inflight promise.

if it were a string, we will still need another variable to keep track of any inflight promises to  `.rawChainSpecPath(...)`

Reply:
BTW, this to cache and dedupe invocations to `.rawChainSpecPath(...)`

Reply:
I might have misunderstood let me do some testing locally

Reply:
without this change, `console.log` inside `createRawChainSpec` shows that the relay chain chain-spec is created more than once for networks with parachains